source/documentation/10-switching-to-production.md

```diff
@@ -2,9 +2,27 @@
 
 Once you have finished testing with your sandbox account, here are the steps you will need to take to switch over from testing to live.
 
-1. You’ll need to request a live account by emailing GOV.UK Pay at govuk-pay-support@digital.cabinet-office.gov.uk. Once you have a live account, it will appear in the “My Services” section and be clearly labelled as a “Live” account, e.g. “worldpay live”, “smartpay live” or “ePDQ live”.
+## Set up GOV.UK Pay account
 
-2. Generate an API key for use with your live code; refer to the documentation to see how to do this.
+Request a live account by emailing GOV.UK Pay at [govuk-pay-support@digital.cabinet-office.gov.uk](govuk-pay-support@digital.cabinet-office.gov.uk). Once you have a live account, it will appear in the “My Services” section and be clearly labelled as a “Live” account, e.g. “worldpay live”, “smartpay live” or “ePDQ live”.
+
+To set up your live account credentials:
+
+1. Go to the [GOV.UK Pay admin site](https://selfservice.payments.service.gov.uk/login).
+1. Sign in to your GOV.UK Pay account.
+1. Go to _My Services_ and click on the _ePDQ Live_ service.
+1. Go to _Settings > Account Credentials > Edit credentials_.
+1. Complete the fields on this page:
+  - _PSP ID_ - enter your PSP ID for ePDQ
+  - _Username_ - enter the API user’s username
+  - _Password_ - enter the API user’s password
+  - _SHA-IN passphrase_ - enter the SHA-IN passphrase created on the Data and origin verification page
+  - _SHA-OUT passphrase_ - enter the SHA-OUT passphrase created on the Transaction feedback page
+1. Click _Save credentials_ to go back to the _Account Credentials_ page.
+
```

Comment:
this doesn't quite seem to be displaying correctly, at least as far as github markdown's parser is concerned
<img width="679" alt="screen shot 2017-10-18 at 12 10 16" src="https://user-images.githubusercontent.com/1420504/31715883-77fe8aaa-b3fd-11e7-97ac-793ade20804d.png">

Reply:
It did the same thing when I first created the content, but then when it went live it was ok. I think it is the github parser.

Reply:
ok cool

Reply:
I've removed the internal links for now. Are you happy to approve this PR?